src/cljs/main/broadfcui/page/workspace/summary/catalog.cljs

```diff
@@ -91,128 +94,110 @@
+                                      (when (= val current-value) {:checked true}))]
+                             [:div {:style {:padding "0 0.4em" :fontWeight "500"}} (or label (str val))]])]
+             (when-not (:hidden prop)
+               [(if enumerate :li :div) {}
+                [:div {:style {:marginBottom 2}}
+                 (:title prop)
+                 (when consentCode
+                   (list
+                    " ["
+                    [:abbr {:style {:cursor "help" :whiteSpace "nowrap" :borderBottom "1px dotted"}
+                            :title (get-in library-schema [:consentCodes (keyword consentCode)])}
+                     consentCode]
+                    "]"))
+                 (when required?
+                   [:span {:style (colorize {:fontWeight "bold"})}
+                    " (required)"])]
+                (cond enum
+                      (if (< (count enum) 4)
+                        [:div {:style {:display "inline-block" :margin "0.75em 0 0.75em 1em"}}
+                         (map #(checkbox {:val %}) enum)]
+                        (style/create-identity-select {:value (or current-value ENUM_EMPTY_CHOICE)
+                                                       :style (colorize {})
+                                                       :onChange update-property}
+                                                      (cons ENUM_EMPTY_CHOICE enum)))
+                      (= type "boolean")
+                      [:div {:style {:display "inline-block" :margin "0.75em 0 0.75em 1em"}}
+                       (checkbox {:val true :label (case wording "yes/no" "Yes" "True")})
+                       (checkbox {:val false :label (case wording "yes/no" "No" "False")})
+                       (when-not required?
+                         (checkbox {:val nil :label (or emptyChoice "N/A")}))]
+                      (= datatype "freetext")
+                      (style/create-text-area {:style (colorize {:width "100%"})
+                                               :value current-value
+                                               :onChange update-property
+                                               :rows 3})
+                      (= (:typeahead prop) "ontology")
+                      [:div {:style {:marginBottom "0.75em"}}
+                       [comps/Typeahead {:field-attributes {:placeholder "Select an ontology value."
+                                                            :style (colorize {:width "100%" :marginBottom "0px"})
+                                                            :value current-value
+                                                            :onChange update-property}
+                                         :remote {:url (str (config/api-url-root) "/duos/autocomplete/%QUERY")
+                                                  :wildcard "%QUERY"
+                                                  :cache false}
```

Comment:
this may be more of a question for sarah - but maybe the cache here should be false? we aren't expecting the possible options to be very dynamic...